terracumber_config/tf_files/aws_mirror.tf

```diff
@@ -1,7 +1,7 @@
 // Mandatory variables for terracumber
 variable "URL_PREFIX" {
   type = string
-  default = "https://ci.suse.de/view/Manager/view/Manager-4.3/job/SUSEManager-4.3-AWS"
+  default = "https://ci.suse.de/view/Manager/view/Manager-5.0/job/SUSEManager-5.0-AWS"
```

Comment:
Actually, we can use this for both versions right? Maybe what we need is an independent pipeline.

Reply:
Done